extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/RequestMetricInfo.java

```diff
@@ -85,12 +85,23 @@ protected static String normalizePath(String uri) {
         if (uri == null || uri.isEmpty() || ROOT.equals(uri)) {
             return ROOT;
         }
-        // Label value consistency: result should begin with a '/' and should not end with one
-        String workingPath = MULTIPLE_SLASH_PATTERN.matcher('/' + uri).replaceAll("/");
-        workingPath = TRAILING_SLASH_PATTERN.matcher(workingPath).replaceAll("");
```

Comment:
Looks like you could drop the static fields?

Reply:
True.